quickjs.c

```diff
@@ -51816,6 +51816,31 @@ void JS_AddPerformance(JSContext *ctx)
     JS_FreeValue(ctx, performance);
 }
 
+/* Equality comparisons and sameness */
+JS_BOOL JS_IsEqual(JSContext *ctx, JSValue op1, JSValue op2)
```

Comment:
The return type **must be `int`** because there are 3 possible return values: -1, 0 and 1